examples/native-demo/src/main/resources/META-INF/native-image/jni-config.json

```diff
@@ -0,0 +1,14 @@
+[
+{
+  "name":"java.lang.String",
+  "methods":[{"name":"lastIndexOf","parameterTypes":["int"] }, {"name":"substring","parameterTypes":["int"] }]
+},
+{
+  "name":"java.lang.System",
+  "methods":[{"name":"getProperty","parameterTypes":["java.lang.String"] }, {"name":"setProperty","parameterTypes":["java.lang.String","java.lang.String"] }]
+},
+{
+  "name":"org.springframework.boot.loader.JarLauncher",
```

Comment:
I don't see any uses of spring boot in this project?

Reply:
the jar launcher is part of the repacked jar.